src/browser/squirrel-update.coffee

```diff
@@ -21,6 +22,9 @@ fileKeyPath = 'HKCU\\Software\\Classes\\*\\shell\\Atom'
 directoryKeyPath = 'HKCU\\Software\\Classes\\directory\\shell\\Atom'
 backgroundKeyPath = 'HKCU\\Software\\Classes\\directory\\background\\shell\\Atom'
 environmentKeyPath = 'HKCU\\Environment'
+openWithKeyPath = 'HKCU\\Software\\Classes\\Atom'
+filetypeKeyPath = (fileExtension) -> "HKCU\\Software\\Classes\\.#{fileExtension}\\OpenWithProgIds"
+openWithFileExtensions = require path.join(binFolder, 'win32-open-file-extensions.json')
```

Comment:
I think it might be better to keep the `win32-open-file-extensions.json` in the `src/` folder somewhere instead of in `resources/` since it is a `.json` file and not a platform specific file type like a `.plist`.